apps/condo/domains/ticket/schema/TicketExportTask.js

```diff
@@ -143,8 +143,8 @@ const TicketExportTask = new GQLListSchema('TicketExportTask', {
             isRequired: true,
             hooks: {
                 resolveInput: async ({ resolvedData }) => {
-                    const { timeZoneFromUser } = resolvedData
-                    return normalizeTimeZone(timeZoneFromUser) || DEFAULT_ORGANIZATION_TIMEZONE
+                    const { timeZone } = resolvedData
+                    return normalizeTimeZone(timeZone) || DEFAULT_ORGANIZATION_TIMEZONE
```

Comment:
it's better to use the timezone and lang from the organization schema. browser timezones are not reall

Reply:
where are we going to get it from? Organizations do not currently have such data.

Reply:
perhaps it is worth thinking about it within the framework of another task